src/views/DatasetManager/components/DatasetOverview/components/DatasetOverviewPlaceholder/DatasetOverviewPlaceholder.js

```diff
@@ -94,12 +95,23 @@ export const DatasetOverviewPlaceholder = () => {
     if (
       currentDatasetId == null ||
       !currentDatasetIngestionStatus ||
-      ![INGESTION_STATUS.ERROR, INGESTION_STATUS.UNKNOWN].includes(currentDatasetIngestionStatus)
+      ![INGESTION_STATUS.ERROR, INGESTION_STATUS.UNKNOWN, INGESTION_STATUS.PENDING].includes(
+        currentDatasetIngestionStatus
+      )
     )
       return null;
 
     return currentDatasetType === DATASET_SOURCE_TYPE.LOCAL_FILE ? (
       <ReuploadFileDatasetButton datasetId={currentDatasetId} iconButton={false} />
+    ) : currentDatasetIngestionStatus === INGESTION_STATUS.PENDING ? (
+      <Button
+        data-cy="dataset-overview-abort-button"
+        variant="contained"
+        color="error"
+        onClick={() => stopRunner(currentDatasetId)}
+      >
+        {t('commoncomponents.datasetmanager.overview.placeholder.abortButton', 'Abort')}
+      </Button>
```

Comment:
IIUC, the button will be visible for all datasets with status "pending", even those that are not of ETL/runners type. What happens if we try to call `stopRunner` for a dataset of type ADT (or Azure Storage)? 
Shouldn't we only show the "stop runner" button for datasets of type "ETL"?

Reply:
It's a mess